lib/gem2rpm/package.rb

```diff
@@ -1,4 +1,6 @@
 require 'delegate'
+# require 'rubygems' to avoid NameError at Ruby 1.8.7.
+require 'rubygems' unless defined?(Gem)
```

Comment:
No, it happens with more recent version of RubyGems then we are shipping by default in RHEL 6. I.e. there is RubyGems 2.5.1 used on Travis while we ship RubyGems 1.3.7 with RHEL 6.

But the thing is, that our users are typically using gem2rpm installed via RubyGems. In this case, the /usr/bin/gem2rpm executable generated by RubyGems ensures, that rubygems are loaded prior anything else no matter what is the RubyGems version. On more recent versions os Ruby are RubyGems loaded by default.

IOW by these two lines you are trying to solve scenario which is specific to Travis in this specific configuration and the case when one would run the gem2rpm directly from the checked out repository using obsolete version of Ruby. Nothing worth of the line.